lib/ruby_lsp/requests/completion_resolve.rb

```diff
@@ -0,0 +1,36 @@
+# typed: strict
+# frozen_string_literal: true
+
+module RubyLsp
+  module Requests
+    class CompletionResolve < Request
```

Comment:
We'll need to add documentation similar to what we have for other requests. Don't worry about the demo GIF, we can record that after it's merged. But let's add the docs with the link to the spec.